internal/exec/util/blkid.c

```diff
@@ -201,7 +201,7 @@ static result_t extract_part_info(blkid_partition part, struct partition_info *i
 	// label
 	ctmp = blkid_partition_get_name(part);
 	err = checked_copy(info->label, ctmp, PART_INFO_BUF_SIZE);
-	if (err)
+	if (err && err != RESULT_LOOKUP_FAILED)
```

Comment:
I do have some fear that this is going to leave `info->label` valid but pointing to uninitialized memory. What do you think of putting an explicit NUL terminator there in this case?

Reply:
Indeed, I didn't realize this is the only place where info is initialized. Will fix shortly.

Reply:
Done.

`blkid_partition_get_name` will return `NULL` instead of an empty string if GPT's _PartitionName_ entry is empty (i.e. the first byte is NUL) - so mapping it back to "empty" seems to be the correct solution.